src/components/cc-invoice/cc-invoice.js

```diff
@@ -51,22 +51,22 @@ export class CcInvoice extends LitElement {
 
     return html`
       <cc-block .icon=${iconFile} class=${classMap({ 'has-errors': this.state.type === 'error' })}>
-        <div slot="title">
+        <div slot="header-title">
           ${skeletonNumber ? html` ${i18n('cc-invoice.title')} <span class="skeleton">${number}</span> ` : title}
         </div>
 
         ${this.state.type === 'error'
-          ? html` <cc-notice intent="warning" message="${i18n('cc-invoice.error')}"></cc-notice> `
+          ? html` <cc-notice slot="content" intent="warning" message="${i18n('cc-invoice.error')}"></cc-notice> `
           : ''}
         ${this.state.type !== 'error'
           ? html`
-              <div slot="button">${ccLink(invoice.downloadUrl, i18n('cc-invoice.download-pdf'), skeleton)}</div>
-              <div class="info">
+              <div slot="header-right">${ccLink(invoice.downloadUrl, i18n('cc-invoice.download-pdf'), skeleton)}</div>
+              <div slot="content" class="info">
                 <em class=${classMap({ skeleton })}>
                   ${i18n('cc-invoice.info', { date: invoice.emissionDate, amount: invoice.amount })}
                 </em>
               </div>
-              <cc-html-frame class="frame" ?loading="${skeleton}" iframe-title="${title}">
+              <cc-html-frame slot="content" class="frame" ?loading="${skeleton}" iframe-title="${title}">
```

Comment:
**suggestion:** in responsive, the padding around the invoice frame is gone
preview:
![image](https://github.com/user-attachments/assets/0c63e9c9-d1a8-4720-9126-540a72422bd5)

prod:
![image](https://github.com/user-attachments/assets/6e04f047-ea1a-4ecf-8972-13d8adfc2734)


If you want to restore the padding, the easiest solution would be to slot into `content-body` instead of `content` :thinking: 

We can discuss this in sync if you need, I'm not sure we need the padding anyway, we can do something else.